### DISCUSSION TITLE
Volume control broken after macOS 12.6 upgrade?

### DISCUSSION THREAD

**Discussion by dtsimon** [MonitorControl/MonitorControl#1184] [General]
### Before opening the issue, have you...?

- [X] Searched for existing issues
- [X] Looked through [the wiki](https://github.com/MonitorControl/MonitorControl/wiki)
- [X] Updated MonitorControl to the latest version (if applicable)

### Describe the bug

After the upgrade to macOS 12.6, MC seems to have stopped controlling volume on my Dell U3818DW (latest firmware M3B106). Everything was working fine until today.

### Steps to reproduce

1. Try to adjust volume.
2. Volume OSD appears and scales as it should.
3. Volume can be muted for Spotify, but not for, e.g., sending mail in Mail.app

### Expected behavior

MC should control the volume in all apps and services.

### Environment Information (please complete the following information)

```markdown
- macOS version: 12.6
- Mac model: MacBook Pro (13-inch, 2019, Four Thunderbolt 3 ports)
- MonitorControl version: v4.1.0
- Monitor(s): Dell U3818DW
- Apple Silicon/M1 (yes or no): no
```

**Comment by bkosse** [comment]
same here

- macOS version: 12.6
- Mac model: MacBook Pro (14-inch, 2021)
- MonitorControl version: v4.1.0
- Monitor(s): LG ULTRAWIDE
- Apple Silicon/M1: yes

**Comment by the0neyouseek** [comment]
Hi @dtsimon @bkosse
Checking your monitor osd, does volume change at all and are you using and HDMI or an Usb-c cable ?

**Comment by bkosse** [comment]
Volume does not change at all.
I'm using HDMI.

**Comment by the0neyouseek** [comment]
> Volume does not change at all.
> I'm using HDMI.

On M1/M2 macs HDMI does not allow for DDC (hardware) control (Apple's fault and sadly nothing we can do here) so this is expected behavior. You can try to use an usb-c cable if you want volume control. Beware that a lot of monitor does not support it even if they support DDC control

**Comment by ErinPo** [comment]
Same here.

- macOS version: 12.6
- Mac model: MacBook M1 (13-inch, 2020, two Thunderbolt 3 ports)
- MonitorControl version: v4.1.0   ^136
- Monitor(s): Samsung The Frame
- Apple Silicon/M1 (yes or no): yes
- USB-C to HDMI adapter and HDMI to HDMI cable

I used the same cables on a different model Samsung TV back when MonitorControl 2.0 was the latest and controlling sound worked. Unfortunately, I have not tested prior to macOS 12.6 with MonitorControl v4. I don't use Spotify. Muting does not work for anything. There is no effect on sound. Only the OSD animation. Brightness adjustments work as expected.
